Jim Angel going to emeritus

jimangel opened the pull request:
**TL;DR:** This PR removes myself as an approver/reviewer in SIG Docs.

Ref: https://github.com/kubernetes/website/pull/38864

aojea commented:
😢

MadhavJivrajani commented:
Thank you for everything Jim! ♥️

sftim reviewed (commented):
I do hesitate to write anything except positive feedback here, but: it'd be nice to split the update to add `sig-docs/offboarding.md` into its own commit.

/lgtm
:wave: - you'll be much missed

divya-mohan0209 commented:
Thank you for everything, Jim :)

divya-mohan0209 reviewed (commented):
I've left an initial set of comments on the offboarding docs.  However, I second Tim on separating it into an individual commit.

a-mccarthy commented on sig-docs/offboarding.md:
```diff
@@ -0,0 +1,24 @@
+# Offboarding a SIG Co-chair (Emeritus)
+
+After serving more than 6 months, co-chairs should prioritize finding successors, and moving to emeritus, a normal part of fostering a healthy SIG docs community.
```
Maybe its worth clarifying here that your term isn't 6 months, but it should be a priority to finding and mentoring folks who will one day could take your place as chair when you are ready to step down or hit the term limit.

cblecker commented:
/approve
/hold

Approving for top level. Please remove hold once feedback is addressed or is deemed unnecessary, and this is ready for merge.

jberkus commented:
All:

So ... what's going on with this PR?

jimangel commented:
> So ... what's going on with this PR?

Apologies @jberkus, that's on me. I let it rot. Addressing feedback today for merging.

jimangel commented:
My sincerest apologies for how long this took to resolve. I hope today is the first step forward to a better response time from me 😅

- I addressed merge conflicts from my old branch
- @sftim I split the PR into 2 commits (one for the onboarding docs and one for emeritus).
- @divya-mohan0209 and @a-mccarthy I removed the `(optional)` steps from the bullets and softened the language around `6 months` served before looking at replacements.

I think it's good to merge now. The off-boarding doc isn't meant to be authoritative, but more of a "something is better than nothing." I think once #6952 is figured out, it can be updated or merged into something more "official."

Thanks all and sorry again for how long this took to address feedback - not a reputation I want to keep. I'll catch y'all around! Cheers!

reylejano commented:
Thank you @jimangel 

/lgtm
/unhold
